modules/nf-core/biscuit/mergecg/main.nf

```diff
@@ -23,6 +24,7 @@ process BISCUIT_MERGECG {
     def args2 = task.ext.args2 ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
     """
+    set -o pipefail
```

Comment:
nf-core pipelines already have this, cf https://github.com/nf-core/tools/blob/master/nf_core/pipeline-template/nextflow.config#L175